### ISSUE TITLE
Equation Report

### ISSUE THREAD

**Issue by koay9f** [ORNL-AMO/VERIFI#1306]
Small report that has the equations for each analysis group: string, R2, adj R2, model p-value, notes
Include any invalidity or SEP failures.  If SEP failure - include the SEP table

At corporate level, provide for all facilities

**Comment by koay9f** [comment]
Duplicate of #1552?

**Comment by rmroot** [comment]
We do have an analysis report now that has all these details and it can be customized to just show these specific details

**Comment by koay9f** [comment]
Agreed.  I think the big thing here is that this is a company level report and is arranged a bit differently...
So I would do it so this was in a big table where group is a column that they can sort by (so I can see all "Electricity" near each other), but all this info in 1 table for all groups in all facilities 
The model notes would be the pre-generated ones that say things like "no production variable" and "failed validation"
<img width="1821" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd2dlQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--82863837c92bd7136e8b32ccccb32fe7d20d6526/image.png" alt="image.png" />


then have all of these kind of things in a "individual group reports" toggle like we have in a few other places
<img width="1284" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzhlQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1830b83f23dff4a53eefa11dc9999b16b7c03032/image.png" alt="image.png" />
